perceval/utils/matrix.py

```diff
@@ -145,19 +146,35 @@ def random_unitary(n: int, parameters: Optional[Union[np.ndarray,list]] = None)
         :return: a numeric Matrix
         """
         if parameters is not None:
-            assert len(parameters) == 2*n**2, "parameters has not the right size: should be %d, and is %d" % (
-                2*n**2, len(parameters)
-            )
-            a = np.reshape(parameters[:n**2], (n, n))
-            b = np.reshape(parameters[n**2:], (n, n))
-            u = a + 1j * b
+            warnings.warn("use parametrized_unitary(parameters) instead to create a parametrized unitary "
```

Comment:
You request users to call `parametrized_unitary(parameters)` but shouldn't it be `parametrized_unitary(n, parameters)` ?

Reply:
yes